ext/standard/string.c

```diff
@@ -2445,6 +2445,46 @@ PHP_FUNCTION(substr)
 }
 /* }}} */
 
+/* {{{ proto boolean str_begins(string str, string search_value [, case_sensitive = true])
+   Checks if str begins with search_value */
+PHP_FUNCTION(str_begins) {
+	zend_string *str, *search_value;
+	int argc = ZEND_NUM_ARGS();
+	int case_sensitive;
+
+	if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "SS|l", &str, &search_value, &case_sensitive) == FAILURE) 
+		RETURN_NULL();
+
+	for (int i = 0; i < search_value->len; i++) {
```

Comment:
The `int i` should go to the top of the function. That should fix the build issue.